uitools/examples/README.md

```diff
@@ -1,10 +1,9 @@
 # UI Tools Examples
 
-This module contains two example apps, [cpp_quick](cpp_quick/cpp_quick.pro) and 
-[qml_quick](qml_quick/qml_quick.pro).
+This module contains the example app, [cpp_quick](cpp_quick/cpp_quick.pro).
```

Comment:
```suggestion
This module contains the example app, [UitoolsExample](UitoolsExamples.pro).
```

Might want to double check this suggestion is working. I can't click the link it generates through this interface. But `cpp_quick/cpp_quick.pro` doesn't exist anymore.